pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -84,6 +84,12 @@ stdenv.mkDerivation (finalAttrs: {
 
   propagatedNativeBuildInputs = [ lndir ];
 
+  # libQt5Core links calls CoreFoundation APIs that call into the system ICU. Binaries linked
+  # against it will crash during build unless they can access `/usr/share/icu/icudtXXl.dat`.
+  sandboxProfile = lib.optionalString stdenv.isDarwin ''
```

Comment:
Maybe we can use `propagatedSandboxProfile` despite the fact that it doesn't work? Since it still signals intent?